port/unix/omrsysinfo.c

```diff
@@ -4552,6 +4539,7 @@ omrsysinfo_cgroup_subsystem_iterator_next(struct OMRPortLibrary *portLibrary, st
 			}
 		}
 	}
+	metricElement->key = currentElement->metricTag;
```

Comment:
Why write this in the case of an error?

Reply:
To have it handy for caller to display the value for this particular key is `unavailable`.

Reply:
> A cleaner approach is to have a separate call that provides the metric name and one that gets the data.

@DanHeidinga Will be adding an api to get the metric name seperately. Thank you

Reply:
Are there other usecases or pieces of data that need to be available even in error cases?  If so, let's get them described here so the right api can be designed upfront rather than through review comments.

Reply:
@DanHeidinga As of now i haven't came across any requirement for making other pieces of data  available in that context.